src/libraries/Common/src/Microsoft/Win32/SafeHandles/SafeUnicodeStringHandle.cs

```diff
@@ -23,12 +24,12 @@ public SafeUnicodeStringHandle(string s)
         public unsafe SafeUnicodeStringHandle(ReadOnlySpan<char> s)
             : base(IntPtr.Zero, ownsHandle: true)
         {
-            // If s == default then the span represents the null string,
+            // If s is null ref then the span represents the null string,
```

Comment:
Nit:
```suggestion
            // If s contains a null ref then the span represents the null string,
```
Same for other similar cases.